back/db/migrate/20220531123916_create_home_pages.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class CreateHomePages < ActiveRecord::Migration[6.1]
+  def change
+    create_table :home_pages, id: :uuid do |t|
```

Comment:
Will we add fields like `signedOutHeaderOverlayColor` from `back/config/schemas/style.schema.json`? They are on the design too.

Reply:
Ah, I didn't know about this one! Yes will add.

Reply:
I'm also missing the image itself, because I think this is not the AppConfig currently.